tests/func/test_version.py

```diff
@@ -0,0 +1,26 @@
+import re
+
+from dvc.command.version import CmdVersion
+from dvc.cli import parse_args
+
+
+def test_run():
+    cmd = CmdVersion(parse_args(["version"]))
+    ret = cmd.run_cmd()
+    assert ret == 0
+
+
+def test_info(caplog):
+    check_data = ["DVC version", "Python version", "Platform"]
+    cmd = CmdVersion(parse_args(["version"]))
+    cmd.run()
+    info_dict = {}
+
+    for record in caplog.records:
+        msg = re.sub(r"\n", ": ", record.msg)
+        info_list = msg.split(": ")
```

Comment:
By using regex, I meant something like this:

```python
def test_info(caplog):
    CmdVersion(parse_args(["version"])).run()

    assert re.search(re.compile(r'DVC version: \d+\.\d+\.\d+'), caplog.text)
    assert re.search(re.compile(r'Python version: \d\.\d\.\d'), caplog.text)
    assert re.search(re.compile(r'Platform: .*'), caplog.text)
```

I liked the other version more :sweat_smile:, this one is hard to read and is not that clear what are you testing.